KuringApp/KuringApp/Subscription/SubscriptionView.swift

```diff
@@ -7,41 +7,71 @@
 
 import Model
 import SwiftUI
+import SubscriptionFeature
 import ComposableArchitecture
 
 struct SubscriptionFeature: Reducer {
     struct State: Equatable {
-        @BindingState var subscriptionType: SubscriptionType = .university
+        var subscriptionType: SubscriptionType = .university
         
-        var myDepartments: IdentifiedArrayOf<Department> = []
+        /// 대학 공지 리스트
+        let univNoticeTypes: [String] = ["학사", "취창업", "국제", "장학", "입학", "학생", "산학", "일반"]
+        /// 대학 공지 리스트 중 내가 구독한 공지
+        var selectedunivNoticeType: [String] = []
```

Comment:
lowerCamelCase로 변경되면 좋을 것 같아요!
```suggestion
        var selectedUnivNoticeType: [String] = []
```

Reply:
수정 완료했습니다